### ISSUE TITLE
The WYSIWYG editor disappears when the styles are recompile with gulp all

### ISSUE THREAD

**Issue by fstigre** [TypiCMS/Base#92]
First of all let me start by saying that I really like the CMS. Keep up the good work!

For some reason the WYSIWYG editor in the Pages section (on each page) disappears when the styles are re-compiled with _gulp all_.

Any idea what could be causing this issue?

Here is a screenshot of the Terms of use page in a fresh install and after _gulp all_ . 

FYI - I tried this twice and when you first install TypiCMS the editor is visible but it disappears as soon as Gulp recompiles.

![screen shot 2017-03-05 at 7 59 32 am](https://cloud.githubusercontent.com/assets/1509739/23587895/c4a2cdfc-017a-11e7-8fef-ba6c9332856f.png)

Here is a screenshot of a clean-working editor
![screen shot 2017-03-05 at 8 25 13 am](https://cloud.githubusercontent.com/assets/1509739/23588070/6834b9b0-017d-11e7-9815-54bd23990537.png)

**Comment by fstigre** [comment]
I did a little research and the issue seem to be coming from the plugin.js file in the widget folder.

> public/components/ckeditor/plugins/widget/plugin.js

My temporary fix was to comment the 'ckeditor' line in the gulpfile.js
```
gulp.task('all', [
    'less-public',
    'less-admin',
    'js-public',
    'js-admin',
    'fonts',
    'angular-locales',
    'fancybox-img',
   //  'ckeditor',
    'watch'
]);
```

**Comment by sdebacker** [comment]
Hello,

Add 'widgetselection' to the plugins list in the task "ckeditor" in gulpfile.js
I just released v2.9.3 that fix it.

Samuel

**Comment by fstigre** [comment]
Added 'widgetselection' to the plugins list and it temporarily fixed the issue but then it came back after a few compiles.
